.github/workflows/release.yml

```diff
@@ -52,3 +52,37 @@ jobs:
         run: npm run release:phase5
         env:
           NODE_AUTH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+  start-deployment:
+    needs: release
+    runs-on: [coveo, linux, x64, ec2]
```

Comment:
`runs-on: [coveo]` === self hosted ?

Reply:
Yes sir 👍
Required to trigger the deployment pipeline